driver-async/src/test/functional/com/mongodb/async/client/Fixture.java

```diff
@@ -65,7 +69,14 @@ public static MongoClientSettings getMongoClientSettings() {
 
     public static com.mongodb.MongoClientSettings.Builder getMongoClientBuilderFromConnectionString() {
         com.mongodb.MongoClientSettings.Builder builder = com.mongodb.MongoClientSettings.builder()
-                .applyConnectionString(getConnectionString());
+                .applyConnectionString(getConnectionString())
```

Comment:
I don't think we need to make this change in the global MongoClient.  Does it help?

Reply:
I reverted this back to its original state.